warehouse/query-core/src/main/java/datawave/query/tables/ssdeep/FullSSDeepDiscoveryChainStrategy.java

```diff
@@ -60,11 +60,7 @@ public Iterator<DiscoveredSSDeep> runChainedQuery(AccumuloClient client, Query i
                     Iterator<ScoredSSDeepPair> initialQueryResults, QueryLogic<DiscoveredSSDeep> latterQueryLogic) throws Exception {
         final Iterator<DiscoveredSSDeep> it = super.runChainedQuery(client, initialQuery, auths, initialQueryResults, latterQueryLogic);
 
-        // Create a defensive copy of the score map because stream evaluation may be delayed.
-        final Multimap<String,ScoredSSDeepPair> localScoredMatches = TreeMultimap.create(String.CASE_INSENSITIVE_ORDER, ScoredSSDeepPair.NATURAL_ORDER);
-        localScoredMatches.putAll(scoredMatches);
```

Comment:
I should have picked up the need to do this as a sign that I had a concurrency issue. Thanks for the analysis.